opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -1116,6 +1121,36 @@ private static void pauseToAwaitProfiler() {
         }
     }
 
+    private static String getCtagsCommand() {
+        StringBuilder result = new StringBuilder();
+        Ctags ctags = CtagsUtil.newInstance(env);
+        List<String> argv = ctags.getArgv();
+        for (int i = 0; i < argv.size(); ++i) {
+            if (i > 0) {
+                result.append("\t");
+            }
+            String arg = argv.get(i);
+            if (arg == null) {
+                result.append("UNDEFINED");
```

Comment:
what will this do ? and how could it happen actually ?

Reply:
I wrote that when `Ctags` initially had a `setBinary()` method and started with `null` binary. I retired that method in a subsequent commit but didn't remove the `"UNDEFINED"` handling.